docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,49 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+[discrete]
+[[esql-ccs-security-model-api-key]]
+===== API key authentication
```

Comment:
Can you also add a section for "TLS certificate authentication". Probably just a small paragraph that links to https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters-cert.html and mentions that same named role must exist on in both clusters and it is recommended that the same named roles across clusters have identical privileges.  Don't want to restate too much here, mostly just calling out that 2 models exist.

Reply:
Makes sense 👍